NOTES.WIN

```diff
@@ -55,62 +82,36 @@
  is, of course, to choose a different set of directories by using
  --prefix and --openssldir when configuring.
 
- GNU C (Cygwin)
- --------------
-
- Cygwin implements a Posix/Unix runtime system (cygwin1.dll) on top of the
- Windows subsystem and provides a bash shell and GNU tools environment.
- Consequently, a make of OpenSSL with Cygwin is virtually identical to the
- Unix procedure.
-
- To build OpenSSL using Cygwin, you need to:
-
- * Install Cygwin (see https://cygwin.com/)
-
- * Install Cygwin Perl and ensure it is in the path. Recall that
-   as least 5.10.0 is required.
-
- * Run the Cygwin bash shell
-
- Apart from that, follow the Unix instructions in INSTALL.
-
- NOTE: "make test" and normal file operations may fail in directories
- mounted as text (i.e. mount -t c:\somewhere /home) due to Cygwin
- stripping of carriage returns. To avoid this ensure that a binary
- mount is used, e.g. mount -b c:\somewhere /home.
-
- It is also possible to create "conventional" Windows binaries that use
- the Microsoft C runtime system (msvcrt.dll or crtdll.dll) using MinGW
- development add-on for Cygwin. MinGW is supported even as a standalone
- setup as described in the following section. In the context you should
- recognize that binaries targeting Cygwin itself are not interchangeable
- with "conventional" Windows binaries you generate with/for MinGW.
+ mingw and mingw64
+ =================
 
+ * MSYS shell and development environment installation:
 
- GNU C (MinGW/MSYS)
- ------------------
+   Download MSYS2 from https://msys2.github.io/ and follow installation
+   instructions. Once up and runinng install even make, perl, [git,]
```

Comment:
s/runinng/running/

Reply:
BTW, what do you think of [git,] here?

Reply:
I would make that `(git if you need it)`